process/sync/baseSync.go

```diff
@@ -1238,6 +1243,29 @@ func (boot *baseBootstrap) handleTokensSuppliesRepopulation() error {
 	return tokensSuppliesProc.SaveSupplies()
 }
 
+func (boot *baseBootstrap) syncAccountsDBs(key []byte, id string) error {
+	// TODO: refactor this in order to avoid treatment based on identifier
```

Comment:
I do not think this is a huge problem - as it only starts the sync - thus this check is done only once.

Reply:
Right, this was copy-pasted from original meta code, wanted to leave the TODO as it was